kong/test_utils.go

```diff
@@ -32,12 +29,23 @@ func RunWhenKong(t *testing.T, versionRange string) {
 	if err != nil {
 		t.Error(err)
 	}
+	return currentVersion
+}
+
+// RunWhenKong skips the current test if the version of Kong doesn't
+// fall in the versionRange.
+// This helper function can be used in tests to write version specific
+// tests for Kong.
+func RunWhenKong(t *testing.T, versionRange string) {
+	t.Helper()
+
+	currentVersion := GetVersionForTesting(t)
 	r, err := NewRange(versionRange)
 	if err != nil {
 		t.Error(err)
 	}
 	if !r(currentVersion) {
-		t.Skipf("kong version %s not in range %s", version, versionRange)
+		t.Skipf("kong version %s not in range %s", currentVersion.String(), versionRange)
```

Comment:
Nit: 

No need to call `.String()` when using the `%s` formatting string.

```suggestion
		t.Skipf("kong version %s not in range %s", currentVersion, versionRange)
```